# HG changeset patch
# User lbourges
# Date 1495051511 -7200
#      Wed May 17 22:05:11 2017 +0200
# Branch zulu8-default
# Node ID 00cb53d6194d412eb7d2dba44564eaa0efabaa3a
# Parent  110a089786f4bc74aeb7b42bb275eb0b6bd06c7c
8180055: Upgrade the Marlin renderer in Java2D (update service list)
Summary: added the double-precision variant + MarlinFX backports + Improved MarlinTileGenerator + higher precision of the cubic / quadratic curve
Reviewed-by: flar, pnarayanan

diff --git a/src/share/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine b/src/share/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
--- a/src/share/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
+++ b/src/share/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
@@ -3,3 +3,4 @@
 
 #  Marlin Rendering Engine module
 sun.java2d.marlin.MarlinRenderingEngine
+sun.java2d.marlin.DMarlinRenderingEngine
diff --git a/src/solaris/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine b/src/solaris/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
--- a/src/solaris/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
+++ b/src/solaris/classes/sun/java2d/pisces/META-INF/services/sun.java2d.pipe.RenderingEngine
@@ -6,3 +6,4 @@
 
 #  Marlin Rendering Engine module
 sun.java2d.marlin.MarlinRenderingEngine
+sun.java2d.marlin.DMarlinRenderingEngine
